ca_tools/launch/joy_teleop.launch

```diff
@@ -1,8 +1,10 @@
 <?xml version="1.0" ?>
+<!-- USE MOVE_ROBOT VARIABLE TO SELECT WHICH ROBOT TO CONTROL, DEFAULT=1 -->
```

Comment:
These changes don't match with the ones that you did in the other launch file.

Reply:
Adressed